src/main/java/ch/njol/skript/expressions/ExprYawPitch.java

```diff
@@ -35,118 +39,182 @@
 import org.bukkit.util.Vector;
 
 @Name("Yaw / Pitch")
-@Description("The yaw or pitch of a location or vector.")
+@Description({"The yaw or pitch of a location or vector.",
+		"A yaw of 0 or 360 represents the positive x direction. Adding a positive number to the yaw of a player will rotate it counter-clockwise.",
+		"A pitch of 90 represents the positive y direction, or upward facing. A pitch of -90 represents downward facing. Adding a positive number to the pitch will rotate the player upwards."})
```

Comment:
```suggestion
@Description({
		"The yaw or pitch of a location or vector.",
		"A yaw of 0 or 360 represents the positive x direction. Adding a positive number to the yaw of a player will rotate it counter-clockwise.",
		"A pitch of 90 represents the positive y direction, or upward facing. A pitch of -90 represents downward facing. Adding a positive number to the pitch will rotate the player upwards."
})
```

Reply:
I was copying the old code so not sure if this is the 'correct' way. I like to keep consistency so I'll look around in some other syntax classes to see how it's handled there.

Reply:
This is the new style for multiline examples/description, it's more readable. New/updated classes uses this style.